src/app/manifest/selectors.ts

```diff
@@ -1,6 +1,19 @@
+import { destinyVersionSelector } from 'app/accounts/selectors';
 import { RootState } from 'app/store/types';
+import { useSelector } from 'react-redux';
 
 export const destiny2CoreSettingsSelector = (state: RootState) =>
   state.manifest.destiny2CoreSettings;
 
+const d1ManifestSelector = (state: RootState) => state.manifest.d1Manifest;
 export const d2ManifestSelector = (state: RootState) => state.manifest.d2Manifest;
+
+export const manifestSelector = (state: RootState) =>
+  destinyVersionSelector(state) === 2 ? d2ManifestSelector(state)! : d1ManifestSelector(state);
```

Comment:
Why the `!` on one and not the other?

Reply:
unintentional, just a staggered replacement of its 2 parts. thanks